### PR TITLE
Model update schema v2

### PR DESCRIPTION
**Pull request by marnikdenouden**
# Update database model
Reflect the changes in database schema version v2.1

### CODE REVIEW

**Review by Doodshoofdaapje — COMMENTED** [review]
love it

File: backend/api/templates/email_template.html [hunk 1]
```diff
@@ -3,6 +3,5 @@
 
 Please click on the link below to confirm you registration:
 
-<!--  Creates unique url including user id and encrypted token -->
```
Comment (andreitudorp) [inline backend/api/templates/email_template.html:6]: Why did you remove comments?

File: backend/api/templates/email_template_confirm_submission.html [hunk 2]
```diff
@@ -3,6 +3,5 @@
 
 Please click on the link below to confirm your submission:
 
-<!--  Creates unique url including submission id and encrypted token -->
```
Comment (andreitudorp) [inline backend/api/templates/email_template_confirm_submission.html:6]: Comment removed?

File: backend/api/admin.py [hunk 3]
```diff
@@ -24,8 +24,8 @@
 admin.site.register(BenchmarkInstance) # Instance table
 # admin.site.register(ProblemCategory) # Category table
 # admin.site.register(SpecifiedProblem) # Problem table
```
Comment (andreitudorp) [inline backend/api/admin.py:26]: Remove unused code

**Review by andreitudorp — COMMENTED** [review]
First comments

File: backend/api/migrations/0036_alter_metric_unit.py
Comment (andreitudorp) [inline backend/api/migrations/0036_alter_metric_unit.py:1]: REMOVE THEM

Comment (andreitudorp) [reply backend/api/migrations/0036_alter_metric_unit.py:1]: :)

**Review by andreitudorp — COMMENTED** [review]
More comments

**Review by andreitudorp — APPROVED** [review]
LGTM! Pls test and fix Ruff!

File: backend/api/fixtures/0001_initial.json [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+    }
+},
+{
+    "model": "api.problemcategory",
+    "pk": "2587bee1-8210-4d2e-a2ff-08a82fce5fc3",
+    "fields": {
+        "name": "Time Slotting Problem",
+        "style": 0,
+        "type": 1,
+        "description": "The Time Slotting Problem (TSP) entails optimizing the allocation of entities into predefined time slots, considering constraints and objectives such as maximizing resource utilization or satisfaction metrics. Commonly encountered in scheduling scenarios, TSP aims to efficiently arrange tasks or events to minimize conflicts and satisfy objectives within a given time frame.",
```
Comment (ThijsEgberts) [inline backend/api/fixtures/0001_initial.json:197]: chatgpt has been cooking too much, please remove this

File: backend/api/fixtures/0001_initial.json [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+        "simulator": null,
+        "validator": null
```
Comment (ThijsEgberts) [inline backend/api/fixtures/0001_initial.json:199]: please add these to the fixture

File: backend/api/fixtures/0001_initial.json [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+{
+    "model": "api.problemcategory",
+    "pk": "c3be0151-e61e-470b-bba6-4b7f26903d3c",
+    "fields": {
+        "name": "Traveling Salesman Problem",
+        "style": 1,
+        "type": 0,
+        "description": "The Traveling Salesman Problem (TSP) is a classic optimization challenge that seeks to determine the shortest possible route for a salesman to visit a given set of cities exactly once and return to the original city. It is an NP-hard problem in combinatorial optimization, making it difficult to solve efficiently as the number of cities increases. TSP has significant practical applications in logistics, manufacturing, and various fields requiring route planning and resource allocation. The complexity arises from the factorial growth of possible routes, necessitating sophisticated algorithms and heuristics to approximate optimal solutions within reasonable computational timeframes.",
+        "simulator": null,
+        "validator": null
```
Comment (ThijsEgberts) [inline backend/api/fixtures/0001_initial.json:211]: please add these

File: backend/api/fixtures/0001_initial.json [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+    "pk": "6d28f80e-d531-4c51-90f8-b0a76b768f20",
+    "fields": {
+        "user": "9a2636e0-984f-4e79-bc9e-13a628f3cf7b",
+        "problem": "3e44afea-c625-4cb9-ad31-41ea0fbd0745",
+        "name": "Example time slotting submission",
+        "created_at": "2024-06-10T13:06:03.824Z",
+        "is_verified": true,
+        "is_downloadable": true
+    }
+},
```
Comment (ThijsEgberts) [inline backend/api/fixtures/0001_initial.json:263]: Should we add a system account for example submissions or something?

**Review by ThijsEgberts — APPROVED** [review]
